<launch>

  <node pkg="valve_tracker_triton" type="valve_tracker_node" name="valve_tracker" output="screen">
    <param name="opening_element_size" value="1"/>
    <param name="closing_element_size" value="0"/>
    <param name="epipolar_width_threshold" value="3"/>
    <param name="binary_threshold" value="80"/>
    <param name="min_value_threshold" value="110"/>

    <!-- Load model -->
    <rosparam command="load" file="$(find valve_tracker_triton)/etc/model.yaml" />

    <remap from="stereo" to="stereo_down"/>
    <remap from="image" to="image_rect_color"/>

  </node>


</launch>
